gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -38,6 +38,7 @@ public class BaselineErrorProneExtension {
             "CompileTimeConstantViolatesLiskovSubstitution",
             "ConsistentLoggerName",
             "ConsistentOverrides",
+            "DefaultLocale",
```

Comment:
Initially, let’s make this check a suggestion, and not add it to this list. We can spin up an excavator to roll out the suggested fix asynchronously, otherwise some baseline upgrades will be blocked on a way that we aren’t currently prepared to handle.

Reply:
On us to build a better ratcheting rollout mechanism in the medium term

Reply:
Yeah it feels a bit pointless to write these error-prone checks if we're too worried about blocking upgrades to actually enforce them.

Reply:
It depends on the criticality of the finding. I suspect this will find a lot of things that aren’t going to cause issues in practice. I agree we should fix them, but we have other work in flight that isn’t worth risking.